Kickstarter-iOS/Tests/Views/ProjectPamphletContentViewControllerTests.swift

```diff
@@ -32,7 +32,7 @@ internal final class ProjectPamphletContentViewControllerTests: TestCase {
 
   override func tearDown() {
     AppEnvironment.popEnvironment()
-    UIView.setAnimationsEnabled(true)
+    UIView.setAnimationsEnabled(false)
```

Comment:
i thought it was interfering with something, but it wasn't. i can change back, but i am also curious, why do we set to `true` when the `setUp` sets them to `false`?